tests/test_helpers/test_profiling.py

```diff
@@ -41,7 +41,6 @@ def output_function(report):
         wait_for_condition(lambda: result, timeout=20)
 
         assert "Profiling details" in result
-        assert "incomplete_to_complete_dialogue_labels" in result
```

Comment:
Now we don't track this specifically, it was something temporal until we found the memory leak. Since now we track the whole gc, if DialogueLabels accumulate, they should show in the most common objects.